setup.py

```diff
@@ -11,6 +11,8 @@
     "xmltodict",
     "six",
     "werkzeug",
+    "sure",
```

Comment:
I had to add these two lines, as these packages were not installed on my machine. To avoid relying on global packages, maybe we should consider setting up a Python virtualenv at some point? http://docs.python-guide.org/en/latest/dev/virtualenvs/